cfme/utils/events_db.py

```diff
@@ -90,11 +90,12 @@ def process_id(self, target_type, target_name):
     def query_miq_events(self, target_type=None, target_id=None, event_type=None, since=None,
                          until=None, from_id=None):
         """Checks whether an event occured.
+
         Args:
             target_type: What kind of object is the target of the event (MiqServer, VmOrTemplate)
             target_id: What is the ID of the object (or name, see :py:meth:`process_id`).
             event_type: Type of the event. Ideally one of the :py:meth:`all_event_types` but other
-                kinds of events exist too.
+                        kinds of events exist too.
```

Comment:
**QUESTION:** Are we sure want to do that?   In my old life I would make the colons aligned and what you did makes sense (but pep doesn't like that), but with the colons the place to align the wrapped text becomes different for each arg.   Maybe this is better to read your way though, even if its different on each arg.   Is there a standard that requires/suggests it be this way?